Source/Scene/MetadataEntity.js

```diff
@@ -91,7 +102,7 @@ MetadataEntity.prototype.setProperty = function (propertyId, value) {
  * Returns a copy of the value of the property with the given semantic.
  *
  * @param {String} semantic The case-sensitive semantic of the property.
- * @returns {*} The value of the property or <code>undefined</code> if the property does not exist.
+ * @returns {*} The value of the property or <code>undefined</code> if the entity does not have this property.
```

Comment:
(anywhere in the code where there's a `getPropertyBySemantic`) should the comment say "have this property" or "have this semantic"?